src/content/plugins/split-chunks-plugin.md

```diff
@@ -159,6 +160,24 @@ When the chunk has a name already, each part will get a new name derived from th
 
 T> `maxSize` takes higher priority than `maxInitialRequest/maxAsyncRequests`. Actual priority is `maxInitialRequest/maxAsyncRequests < maxSize < minSize`.
 
+Note that setting the value for `maxSize` sets the value for both `maxAsyncSize` and `maxInitialSize`.
+
+### `splitChunks.maxAsyncSize`
+
+`number`
+
+Like `maxSize`, `maxAsyncSize` can be applied globally (`optimization.splitChunks.maxAsyncSize`), to cacheGroups (`optimization.splitChunks.cacheGroups[x].maxAsyncSize`), or to the fallback cache group (`optimization.splitChunks.fallbackCacheGroup.maxAsyncSize`).
```

Comment:
```suggestion
Like `maxSize`, `maxAsyncSize` can be applied globally (`splitChunks.maxAsyncSize`), to cacheGroups (`splitChunks.cacheGroups.{cacheGroup}.maxAsyncSize`), or to the fallback cache group (`splitChunks.fallbackCacheGroup.maxAsyncSize`).
```

Reply:
We omit optimization further down the examples, also use `splitChunks.cacheGroups.{cacheGroup}` vs `splitChunks.cacheGroups[x]` for consistency